draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
 
-This example illustrates how the artifacts defined in this specification can be
-represented using W3C Verifiable Credentials Data Model as defined in
-[@VC_DATA].
+This example illustrates how to use the artifacts defined in this specification to secure a payload
+that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
+and does not use JSON-LD. The example uses a content type `credential-claims-set+json` defined in [@VC_JWT], Section 3 in a `cty` JOSE Header value.
 
-SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a VC.
+SD-JWT is equivalent to an Issuer-signed W3C Verifiable Credential (W3C VC). Disclosures are sent alongside a W3C VC.
```

Comment:
the equivalence language reads roughly...  i think there are 2 cases:

SD-JWT is a VC where:

1. the verified / decoded claim set is a core data model media type. (with JSON-LD)
2. the verified / decoded claim set JSON. (without JSON-LD)

Both are "W3C VCs" if a mapping exists for 2.

It would be preferable to call them both "W3C Verifiable Credentials", and define a minimal mapping to ensure 2 is valid, with caveats, like we are doing here: https://github.com/w3c/vc-jwt/pull/60

Reply:
I will merge the current language as a starting point. can you please file an issue on this so that we can fix it later?